typescript/infra/config/environments/mainnet3/warp/warpIds.ts

```diff
@@ -12,6 +12,23 @@ export enum WarpRouteIds {
   EthereumVictionETH = 'ETH/ethereum-viction',
   EthereumVictionUSDC = 'USDC/ethereum-viction',
   EthereumVictionUSDT = 'USDT/ethereum-viction',
+  EthereumZircuitPZETH = 'PZETH/ethereum-zircuit',
   InevmInjectiveINJ = 'INJ/inevm-injective',
   MantapacificNeutronTIA = 'TIA/mantapacific-neutron',
 }
+
+// add new warp route ids here if they are supported by the checker tooling, add an entry to warpConfigGetterMap also
+export enum CheckerWarpRouteIds {
```

Comment:
I split this out into a separate enum as not all warp routes are support by the checker yet

Reply:
do we need to have `WarpRouteIds` as well? I'd be happy to just have a single enum that covers the warp route IDs that we want to check

Or is the difference that we may want monitoring on some warp routes but not checking?

Just feeling like the web of warp IDs is already a bit tedious so it'd be nice to not need another place where we write all these long names lol